components/root/root.jsx

```diff
@@ -140,6 +140,7 @@ export default class Root extends React.PureComponent {
         Client4.setAuthHeader = false;
 
         setSystemEmojis(EmojiIndicesByAlias);
+        store.dispatch(migrateRecentEmojis());
```

Comment:
I don't actually know why we're using the global store directly in this file, but this should be passed in from this file's index.js in the `mapDispatchToProps` to connect it to the store. 

I wonder if that might fix the issue you were having with the current user ID being empty?

Reply:
I have moved it to mapDispatchToProps but still it's coming empty